acquia_lift.admin.wizard.inc

```diff
@@ -1704,11 +1713,68 @@ function _acquia_lift_personalize_campaign_wizard_block_single_save($block_conte
 }
 
 /**
+ * Helper submit function to save a single existing element variation.
+ *
+ * @param array $values
+ *   The values from the form state specific to this option set
+ * @param stdClass $option_set
+ *   The option set to be edited.
+ */
+function _acquia_lift_personalize_campaign_wizard_element_single_save($values, $option_set) {
+  $option_set->label = $values['label'];
+  foreach ($option_set->options as &$option) {
+    $option['option_label'] = $values['options'][$option['option_id']]['option_label'];
+  }
+  personalize_option_set_save($option_set);
+}
+
+/**
+ * Helper submit function to submit the general and advanced information
+ * for an existing option set.
+ *
+ * @param array $values
+ *   The form state values specific to the option set.
+ * @param stdClass $option_set
+ *   The option set object submitted with the form.
+ */
+function _acquia_lift_personalize_campaign_wizard_variations_single_save_general($values, $option_set) {
+  $option_set = personalize_option_set_load($option_set->osid);
```

Comment:
How come the option set needs to be loaded again here, but not in the _acquia_lift_personalize_campaign_wizard_element_single_save() function above?